<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>814</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
    font-family : century gothic ;
	font-size : 24px;
	
}
QFrame {
	background : #333;
	border-radius : 15px;
	background : rgba(0,0,0,0.8);

}
QPushButton{
	border-radius : 15px;
	background : red;
	color : white;
}
QToolButton{
	border-radius : 60px;
	background : red;
}
QLabel{
	color : white;
	background : transparent;
}
QPushButton:hover
{
	border-radius : 15px;
	background : #49ebff;
	color : #333;
}
QLineEdit
{
	background : transparent;
	border : none;
	color : #717072;
	border-bottom : 1px solid #717072;
}
</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>651</width>
     <height>651</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_firstname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Exemple : Mehdi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Last  Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_lastname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>150</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Exemple : Ziadi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone Num</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_phonenumber">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Exemple : 54443324</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>250</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_email">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Exemple : mehdi.ziadi.28@gmail.com</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>400</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_username">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>400</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Exemple : Schorakbi</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>450</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>450</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Exemple : nicetry123456</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_submit">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>540</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_adress">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Adress</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_adress">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Exemple : 8 rue de Breme</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_birthdate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Birth Date</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_birthdate">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>350</y>
      <width>431</width>
      <height>22</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Exemple : 22.05.2001</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
